### PR TITLE
Update Dockerfile for 1.5.0

### PR DESCRIPTION
**Pull request by mikkelhegn**

### CODE REVIEW

File: spin/02-json-api.md [hunk 1]
```diff
@@ -144,12 +144,15 @@ function answer(): string {
...
 
-Install any dependencies with `npm install` and then build and run the Spin app.
+Install the dependencies and then build and run the Spin app.
 
 ```bash
+npm install
 spin build --up
 ```
 
+> Note: `npm install` is only needed when adding new dependencies to the javascript function.
```
Comment (karthik2804) [inline spin/02-json-api.md:154]: It is needed even otherwise to get the `@fermyon/spin-sdk` package.

Comment (mikkelhegn) [reply spin/02-json-api.md:154]: Let me rephrase to "on additional builds".
